### PR TITLE
Little fixes from Jack's PR

### PR DESCRIPTION
**Pull request by lizgzil**
@Jack-Vines made changes in https://github.com/nestauk/ojd_daps_skills/pull/77/files/91a682b6bae3f8cc0b4a7b86847849837a9ea94c to get the package working in OJO. Some of these were done in other PRs, but this should make the rest of them.

@india-kerle - there is a new argument in ExtractSkills "multi_process". This is default to True (as its been set to this before), it shouldn't effect anything and I think we should worry about updating the docs for it.

My version of  `sentence-transformers` was 2.2.2 for some reason anyway, adding this to the requirements.

---

Checklist:

- [ ] I have refactored my code out from `notebooks/`
- [ ] I have checked the code runs
- [ ] I have tested the code
- [ ] I have run `pre-commit` and addressed any issues not automatically fixed
- [ ] I have merged any new changes from `dev`
- [ ] I have documented the code
  - [ ] Major functions have docstrings
  - [ ] Appropriate information has been added to `README`s
- [ ] I have explained the feature in this PR or (better) in `output/reports/`
- [ ] I have requested a code review

### CODE REVIEW

File: ojd_daps_skills/pipeline/extract_skills/extract_skills.py [hunk 1]
```diff
@@ -31,9 +31,18 @@ class ExtractSkills(object):
...
+
     """
 
-    def __init__(self, config_name="extract_skills_toy", local=True, verbose=True):
+    def __init__(
+        self,
+        config_name="extract_skills_toy",
+        local=True,
+        verbose=True,
+        multi_process=True,
```
Comment (india-kerle) [inline ojd_daps_skills/pipeline/extract_skills/extract_skills.py]: should we default to true givens @Jack-Vines 's troubles with it?

**Review by india-kerle — APPROVED** [review]
cool, thanks for this! It looks good to me.

File: ojd_daps_skills/pipeline/extract_skills/extract_skills.py (hunk 1 above)
Comment (lizgzil) [reply ojd_daps_skills/pipeline/extract_skills/extract_skills.py]: good point!

Comment (lizgzil) [reply ojd_daps_skills/pipeline/extract_skills/extract_skills.py]: done

**Pull request merged**